packages/dom/src/dom.js

```diff
@@ -1002,3 +1002,29 @@ export function removeInvalidHTML( HTML, schema, inline ) {
 
 	return doc.body.innerHTML;
 }
+
+/**
+ * Offsets the given rect by the position of the iframe that contains the element.
+ * If the owner document is not in an iframe then it returns with the original rect.
+ *
+ * @param {DOMRect} rect bounds of the element
+ * @param {Document} ownerDocument document of the element
+ *
+ * @return {DOMRect} offsetted bounds
+ */
+export function offsetIframe( rect, ownerDocument ) {
```

Comment:
now that it's a public API, maybe we should find a better name? @ellatrix any thoughts on this?

Reply:
🤔  How about: `offsetBoundsByOwnerFrame`, `offsetRectByOwnerFrame`, `frameToScreenPosition`, `frameToScreenBounds` ?

Reply:
Why does it need to be public? Can we not offset the iframe in the popover after we get the anchor rect? Why are we using `getAnchorRect` in the first place? That's supposed to be an old API to be deprecated at some point.

Reply:
@ellatrix Moved `offsetIframe` back into the popover package. This time into the `utils`.